### PR TITLE
HW-1.3

### PR DESCRIPTION
**Pull request by serg-ksv**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,17 @@
...
+        }
+
+        StringBuilder binStr = new StringBuilder();
+
+        while (value > 0) {
+            binStr.append(value % 2);
+            value /= 2;
+        }
+
+        return binStr.reverse().toString();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: let's remove empty lines

Comment (serg-ksv) [reply src/main/java/core/basesyntax/BinaryString.java]: done

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder binStr = new StringBuilder();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: let's give more informative name for this variable

Comment (serg-ksv) [reply src/main/java/core/basesyntax/BinaryString.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**